templates/basic/README.md

```diff
@@ -0,0 +1,40 @@
+# Welcome to React Router!
+
+- 📖 [React Router docs](https://reactrouter.com/dev)
+
+## Development
+
+Run the dev server:
+
+```shellscript
+npm run dev
+```
+
+## Deployment
+
+First, build your app for production:
+
+```sh
+npm run build
+```
+
+Then run the app in production mode:
+
+```sh
+npm start
+```
+
+Now you'll need to pick a host to deploy it to.
+
+### DIY
+
+If you're familiar with deploying Node applications, the built-in app server is production-ready.
```

Comment:
Is there a way to deploy to CF already?

Reply:
Yeah, I think we ported over the CF packages, so `@react-router/cloudflare` should work.

I haven't messed with it yet and I don't know if we have any documentation, however I'm not sure what if any breaking changes there are